wrappers/gin/gin.go

```diff
@@ -12,18 +18,67 @@ import (
 // TracerKey is the key of the epsagon tracer in the gin.Context Keys map passed to the handlers
 const TracerKey = "EpsagonTracer"
 
+// EpsagonContext creates a context.Background() with epsagon's associated tracer for nexted instrumentations
+func EpsagonContext(c *gin.Context) context.Context {
+	return epsagon.ContextWithTracer(c.Keys[TracerKey].(tracer.Tracer))
```

Comment:
I wonder whether we want to check the tracer key existence and report a custom panic error if it doesn't exist? (bad user usage)
And I would use c.Get(TracerKey) (even though its the same)

Reply:
I feel like addeding a custom panic will not add any value. the default "missing key" panic will be informative enough as it will be pointing here and citing the "EpsagonTracer" key.
This is the reason why I will leave it with Keys.